service/block/event_test.go

```diff
@@ -5,7 +5,7 @@ import (
 	"testing"
 
 	"github.com/celestiaorg/celestia-core/testutils"
-
+	"github.com/celestiaorg/celestia-node/service/header"
 	"github.com/stretchr/testify/require"
```

Comment:
Can someone please enable the linter s.t. we have a canonical order of packages?
```
std 

third party

this repo
```

Reply:
@Bidon15 ?

Reply:
I agree, we need to have this how, checking this ourselves is annoying